### DISCUSSION TITLE
Miner with LGT8F328P-LQFP32 MiniEVB

### DISCUSSION THREAD

**Discussion by alcaspa** [revoxhere/duino-coin#1332] [Ideas]
Hello everyone, I have LGT8F328P-LQFP32 boards that are equivalent to arduino nano v3.0 but when passing the code with Arduino IDE it shows me an error:

#error "ArduinoUniqueID only works on AVR, SAM, SAMD, STM32, Teensy, megaAVR and ESP Architecture"

Is there no way for me to use these boards? 
https://es.aliexpress.com/item/4000151967508.html?gatewayAdapt=glo2esp&spm=a2g0o.order_list.0.0.21ef194dSAcLqu

King Regards from Spain!

**Comment by alberiolima** [comment]
board manager
https://raw.githubusercontent.com/dbuezas/lgt8fx/master/package_lgt8fx_index.json

![Screenshot from 2022-02-15 21-20-26](https://user-images.githubusercontent.com/7103789/154172433-3aaba281-1850-4538-a67e-0c12bfd71c38.png)

**Reply by alcaspa** [reply]
Thank you Alberiolima but I don't know where I have to put that code to make it work with these boards.

**Reply by alberiolima** [reply]
https://support.arduino.cc/hc/en-us/articles/360016119519-How-to-add-boards-in-the-board-manager

**Reply by alberiolima** [reply]
Third Party boards

**Reply by alberiolima** [reply]
https://www.youtube.com/watch?v=1t0vyJ9o7mk

**Reply by alberiolima** [reply]
https://github.com/dbuezas/lgt8fx

**Reply by alcaspa** [reply]
Thank you very much! I am going to try.

**Reply by alberiolima** [reply]
![Screenshot from 2022-02-15 21-34-53](https://user-images.githubusercontent.com/7103789/154173649-e4bf76b3-55d8-42d9-84f9-4ed8a1bd84b9.png)
